lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -2318,6 +2325,29 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     break;
   }
 
+  case Kinded::Kind::FusedRowwiseQuantizedSparseLengthsWeightedSumInstKind: {
+    auto *N = cast<FusedRowwiseQuantizedSparseLengthsWeightedSumInst>(I);
```

Comment:
> `FusedRowwiseQuantizedSparseLengthsWeightedSumInst`

wow these names are not brief

Reply:
Yeah not really happy about it either but unless we want to start using acronyms, not sure how to shorten... 😞

Reply:
FRQSLWS %) :D (good luck to someone reading this)